packages/sdks-tests/src/snippet-tests/editable-regions.spec.ts

```diff
@@ -45,4 +45,21 @@ test.describe('Editable regions in custom components', () => {
     const secondText = await columns.nth(1).textContent();
     expect(secondText?.trim().toLowerCase()).toBe('column 2 text');
   });
+
+  test('should render two builder blocks with content using gen1', async ({
+    page,
+    packageName,
+  }) => {
+    test.skip(!['gen1-react'].includes(packageName));
```

Comment:
why are we adding a separate test only for gen1? why not run the existing one in this file?